### PR TITLE
Us8274 ui

### PR DESCRIPTION
**Pull request by farhadmak**

### CODE REVIEW

File: plugins/ideas/src/components/NewIdea.vue [hunk 1]
```diff
@@ -86,23 +72,11 @@ export default {
...
-      } else {
-        this.setError()
-      }
+    openUrl (url) {
+      window.open(url, '_top')
     },
-    setError () {
-      this.firstStepError = 'Uh oh!'
+    setDone () {
+      this.$v.$touch()
```
Comment (j-rewerts) [inline plugins/ideas/src/components/NewIdea.vue]: What does this do?

File: plugins/ideas/src/components/ViewInitiative.vue [hunk 2]
```diff
@@ -11,6 +11,7 @@
           <div class="md-caption">{{ initiative.createdDate | formatDate }}</div>
         </div>
         <div class="md-body-1 oct-content-block">{{ initiative.description }}</div>
+        <!-- <md-button class="md-raised md-primary"> Clear Form </md-button> -->
```
Comment (j-rewerts) [inline plugins/ideas/src/components/ViewInitiative.vue]: Remove this.

File: plugins/ideas/src/components/ViewInitiative.vue [hunk 3]
```diff
@@ -21,81 +22,47 @@
...
-      <md-dialog-title>Investment Form</md-dialog-title>
-      <md-field>
-        <label>URL</label>
-        <md-input v-model="temp_investmentFormUrl"></md-input>
-      </md-field>
-      <md-button @click="attachInvestmentForm" class="md-primary oct-attach-button">Attach</md-button>
-      <md-progress-bar v-if="invFormLoading" class="md-accent" md-mode="indeterminate"></md-progress-bar>
-    </md-dialog>    
+    </div>
+    <br><br><br><br><br><br><br><br><br>
```
Comment (j-rewerts) [inline plugins/ideas/src/components/ViewInitiative.vue]: GET RID OF THIS!

File: plugins/ideas/src/components/initiative.vue [hunk 4]
```diff
@@ -102,7 +102,8 @@ export default {
       return colors[randIndex]
     },
     openCard (idea) {
```
Comment (j-rewerts) [inline plugins/ideas/src/components/initiative.vue]: If this function is unused, please delete it.

File: plugins/ideas/src/services/ideas/IdeasService.js [hunk 5]
```diff
@@ -16,16 +16,16 @@ let x = class IdeasService {
    * Returns a Promise that resolves with a list of ideas.
    * @returns {Promise} Resolved with an array of ideas.
    */
-  static getIdeas () {
-    return HTTP.get('')
+  static getIdeas (page, pageSize) {
+    return HTTP.get(`?page=${page}&pageSize=${pageSize}`)
```
Comment (j-rewerts) [inline plugins/ideas/src/services/ideas/IdeasService.js]: In the future, set the default values here.

File: plugins/ideas/src/services/service-loader.js [hunk 6]
```diff
@@ -5,6 +5,8 @@
 // Services
 import { IdeasService } from './ideas/IdeasService'
 import { UserService } from './user/UserService'
+// import { StubbedIdeasService } from './ideas/StubbedIdeasService'
```
Comment (j-rewerts) [inline plugins/ideas/src/services/service-loader.js]: Please add this back in.

File: server/docker-compose - Copy.yml [hunk 7]
```diff
@@ -0,0 +1,80 @@
+version: '3'
```
Comment (j-rewerts) [inline server/docker-compose - Copy.yml]: Please delete this file.

File: plugins/ideas/src/components/initiative.vue [hunk 8]
```diff
@@ -24,9 +24,9 @@
     </div>
     
     <md-divider></md-divider>
-
+  
     <md-card-actions>
-      <md-button @click="openUrl" :style="{ color: getColor(initiative) }">View</md-button>
+      <md-button v-on:click="openUrl" :style="{ color: getColor(initiative) }">View</md-button>
```
Comment (farhadmak) [inline plugins/ideas/src/components/initiative.vue]: Use primary colours

File: plugins/ideas/src/components/NewIdea.vue [hunk 9]
```diff
@@ -134,7 +101,9 @@ export default {
         if (idea && idea.url && idea.url.length > 0) {
           this.ideaURL = idea.url
         }
-        this.setDone('first', 'second')
+        this.setDone()
+        // TODO Don't hardcode /you
```
Comment (farhadmak) [inline plugins/ideas/src/components/NewIdea.vue]: Read validation to check for min length

**Pull request merged**